.travis.yml

```
@@ -4,30 +4,24 @@ os:
   - linux
   - osx
 julia:
-  - 0.7
   - 1.0
+  - 1.3
   - nightly
 matrix:
   allow_failures:
     - julia: nightly
   fast_finish: true
-  include:
-    - stage: "Documentation"
-      julia: 1.0
-      os: linux
-      script:
-        - julia --project=docs/ -e 'using Pkg; Pkg.develop(PackageSpec(path=pwd())); Pkg.instantiate(); Pkg.build()'
-        - julia --project=docs/ docs/make.jl
-      after_success: skip
 notifications:
   email: false
-# script:
-#   - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-#   - julia -e 'Pkg.clone(pwd()); Pkg.build("Intervals"); Pkg.test("Intervals"; coverage=true)'
 after_success:
-  - |
-    julia -e '
-      VERSION >= v"0.7.0-DEV.3656" && using Pkg
-      VERSION >= v"0.7.0-DEV.5183" || cd(Pkg.dir("Intervals"))
-      Pkg.add("Coverage"); using Coverage; Codecov.submit(process_folder())
-      Pkg.add("Documenter"); include(joinpath("docs", "make.jl"))'
+  - julia -e 'using Pkg; Pkg.add("Coverage"); using Coverage; Codecov.submit(process_folder())'
+jobs:
```

Only do 32-bit on Linux:
```suggestion
jobs:
  # Test 32-bit only on Linux
  exclude:
    - arch: x86
      os: osx
    - arch: x86
      os: windows
      
```

---

Is there anything we were concerned about being 64-bit specific?

---

My thoughts were to keep this equivalent to how we had it setup before. After an offline discussion I think most of our repos can probably just test against 64-bit Linux and 32-bit Windows. In cases where there are failures we should support a more exhaustive CI matrix via a commit message trigger